src/sentry/static/sentry/app/components/events/interfaces/crashHeader.jsx

```diff
@@ -88,7 +91,17 @@ class CrashHeader extends React.Component {
     const {title, beforeTitle, hideGuide, stackView, stackType, newestFirst} = this.props;
 
     let titleNode = (
-      <h3 className="pull-left">
+      <h3
+        className="pull-left"
+        style={{
+          marginBottom: 0,
+          maxWidth: '100%',
+          width: '100%',
+          marginRight: space(1),
+          marginLeft: space(1),
+          whiteSpace: 'nowrap',
+        }}
+      >
```

Comment:
Yes, I'd do so, but some CSS has a higher specificity because of nested rules. I could not override  the styles..only if I use `!important,` but I'd rather not